ci/docs/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2020-2022, NVIDIA CORPORATION.
+#################################
+# RAPIDS Deployment Docs build script for CI #
+#################################
+
+if [ -z "$PROJECT_WORKSPACE" ]; then
+    echo ">>>> ERROR: Could not detect PROJECT_WORKSPACE in environment"
+    echo ">>>> WARNING: This script contains git commands meant for automated building, do not run locally"
+    exit 1
+fi
+
+export DOCS_WORKSPACE="$WORKSPACE"
+export PROJECTS=(deployment)
+
+make html
+
+#Commit to Website
+cd $DOCS_WORKSPACE
+
+for PROJECT in ${PROJECTS[@]}; do
+    if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
+        mkdir -p api/$PROJECT/$BRANCH_VERSION
+    fi
```

Comment:
```suggestion
    mkdir -p api/$PROJECT/$BRANCH_VERSION
```

I know this was likely copy/pasted from another repo but we can simplify this logic here. `mkdir -p` will still exit with `0` if a folder already exists, so we don't need to guard it in an `if` statement.